RandLAPACK/comps/rl_util.hh

```diff
@@ -513,6 +757,36 @@ bool orthogonality_check(
     return false;
 }
 
+/// Computes an L-2 norm of a given matrix using
+/// 10 steps of power iteration.
+/// Requires additional n * (n + 2) * sizeof(T) bytes of space.
```

Comment:
I don't think this space requirement is true anymore.

Reply:
Fixed. It is also not 10 steps anymore.